src/swarm-lang/Swarm/Language/Capability.hs

```diff
@@ -229,124 +124,23 @@ constCaps = \case
-  Meet -> Just CMeet
-  MeetAll -> Just CMeet
-  Drill -> Just CDrill
   Use -> Nothing -- Recipes alone shall dictate whether things can be "used"
-  Neg -> Just CArith
-  Add -> Just CArith
-  Sub -> Just CArith
-  Mul -> Just CArith
-  Div -> Just CArith
-  Exp -> Just CArith
-  Whoami -> Just CWhoami
-  Self -> Just CWhoami
-  Swap -> Just CSwap
-  Atomic -> Just CAtomic
-  Instant -> Just CGod
-  Time -> Just CTimeabs
-  Wait -> Just CTimerel
-  Scout -> Just CRecondir
-  Whereami -> Just CSenseloc
-  Waypoint -> Just CWaypoint
-  Structure -> Just CStructure
-  Floorplan -> Just CStructure
-  HasTag -> Just CHastag
-  TagMembers -> Just CTagmembers
-  Detect -> Just CDetectloc
-  Resonate -> Just CDetectcount
-  Density -> Just CDetectcount
-  Sniff -> Just CDetectdistance
-  Chirp -> Just CDetectdirection
-  Watch -> Just CWakeself
-  Heading -> Just COrient
-  Key -> Just CHandleinput
-  InstallKeyHandler -> Just CHandleinput
-  Halt -> Just CHalt
-  -- ----------------------------------------------------------------
-  -- Text operations
-  Format -> Just CFormat
-  Concat -> Just CConcat
-  Split -> Just CSplit
-  Chars -> Just CCharcount
-  CharAt -> Just CCode
-  ToChar -> Just CCode
-  -- ----------------------------------------------------------------
   -- Some God-like abilities.
+  Instant -> Just CGod
   As -> Just CGod
   RobotNamed -> Just CGod
   RobotNumbered -> Just CGod
   Create -> Just CGod
   Surveil -> Just CGod
   -- ----------------------------------------------------------------
-  -- arithmetic
-  Eq -> Just CCompare
-  Neq -> Just CCompare
-  Lt -> Just CCompare
-  Gt -> Just CCompare
-  Leq -> Just CCompare
-  Geq -> Just CCompare
-  -- ----------------------------------------------------------------
-  -- boolean logic
-  And -> Just CCond
-  Or -> Just CCond
-  Not -> Just CNegation
-  -- ----------------------------------------------------------------
-  -- exceptions
-  Try -> Just CTry
-  -- ----------------------------------------------------------------
   -- type-level arithmetic
   Inl -> Just CSum
   Inr -> Just CSum
   Case -> Just CSum
+  -- TODO: #563 pair syntax (1,2,3...) should require CProd too
   Fst -> Just CProd
   Snd -> Just CProd
-  -- TODO: #563 pair syntax (1,2,3...) should require CProd too
-
-  -- ----------------------------------------------------------------
-  -- Some additional straightforward ones, which however currently
-  -- cannot be used in classic mode since there is no craftable item
-  -- which conveys their capability. TODO: #26
-  Teleport -> Just CTeleport -- Some space-time machine like Tardis?
-  Appear -> Just CAppear -- paint?
-  Random -> Just CRandom -- randomness device (with bitcoins)?
-  -- ----------------------------------------------------------------
-  -- Some more constants which *ought* to have their own capability but
-  -- currently don't.
-  View -> Nothing -- TODO: #17 should require equipping an antenna
-  Knows -> Nothing
```

Comment:
`view` and `knows` used to require no capabilities, but you (probably inadvertently) removed them, so now it is impossible to execute either of them.  For now we should put them back along with `run` and `use`.

Reply:
Ups, I should learn to move code up instead of Ctrl+X. 😅